### ISSUE TITLE
Can't focus dialog buttons on ComboPopup using iOS VoiceOver

### ISSUE THREAD

**Issue by wlepinski** [ibm-js/deliteful#708]
The user is unable to select the buttons **CANCEL** and **OK** using swipe to change the currently focused element. This is due to the fact that some aria-attributes close the context where the user can change the focus.

The attributes **role**, **aria-expanded**, **aria-owns** and **aria-haspopup** aren't necessary at the root tag.

**Comment by wkeese** [comment]
I tested the third example of ComboPopup.html (on iOS with VoiceOver enabled) and was able to swipe to the OK and Cancel buttons.   What is your test case that fails?
